app/components/search/Search.tsx

```diff
@@ -0,0 +1,144 @@
+import { SearchInput } from "../search/SearchInput";
+import { useSearchParams } from "@remix-run/react";
+import type { Content } from "@prisma/client";
+import { Link, useNavigation, useNavigate } from "@remix-run/react";
+import { useState, useMemo, useEffect } from "react";
+import type { ChangeEvent } from "react";
+
+import { getIconForCategory } from "~/utils";
+const debouce = require("lodash.debounce");
+
+interface SearchProps {
+  searchResult: Content[];
```

Comment:
dette er vel egentlig flertall?
```suggestion
  searchResults: Content[];
```